### ISSUE TITLE
Tests fail for certain random seeds

### ISSUE THREAD

**Issue by Bensk1** [hyrise/hyrise#646]
https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/fix-filename-tests/runs/23/nodes/71/steps/98/log/?start=0

**Comment by Bensk1** [comment]
https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/fix-filename-tests/runs/29/nodes/71/steps/106/log/?start=0

**Comment by Bensk1** [comment]
https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/master/runs/167/nodes/71/steps/106/log/?start=0

**Comment by mrks** [comment]
The first two are 404 because the branches are deleted. Better to copy the output into a gist.

**Comment by Bensk1** [comment]
Can we bring the deleted branches back?

**Comment by Bensk1** [comment]
Apparently not

**Comment by Bensk1** [comment]
https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/fix-clang-tidy/runs/5/nodes/65/steps/112/log/?start=0

Numa Chunk Migration

**Comment by mrks** [comment]
Please back up the text files as Jenkins is quite eager to clean up things:
https://gist.github.com/mrks/ca68506b81a57f92f46694c396c3b88f

**Comment by mrks** [comment]
again here (@lanice)
https://gist.github.com/mrks/b05047dc149ef8127cf4a857ccae82a0

**Comment by lanice** [comment]
Chunk Migration
https://gist.github.com/lanice/dc163763148927af05ebb4b26c789eec

**Comment by mrks** [comment]
```
../../../src/test/operators/export_csv_test.cpp:209: Failure
Value of: file_exists(meta_filename)
  Actual: false
Expected: true
-15
Comparison of file to expected content failed. 
Expected:
1,"Hallo",3.5,12,2.333

Actual:
"xxx"
"www"

"zzz"

../../../src/test/operators/export_csv_test.cpp:210: Failure
Value of: compare_file(filename, "1,\"Hallo\",3.5,12,2.333\n")
  Actual: false
Expected: true
[  FAILED  ] OperatorsExportCsvTest.ExportAllTypes (4 ms)
```

with a seed of 70532

**Comment by timzimmermann** [comment]
[Seed 21328](https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/fix%252F1132-two-column-scans-statistics/runs/7/nodes/65/steps/103/log/?start=0)

```
[ RUN      ] ServerSessionTest.SessionExecutesSimpleQueryCommand
unknown file: Failure

Unexpected mock function call - taking default action specified at:
../../../src/test/server/server_session_test.cpp:66:
    Function call: send_ready_for_query()
          Returns: 16-byte object <B0-0B 00-CC 35-7F 00-00 50-13 00-CC 35-7F 00-00>
Google Mock tried the following 2 expectations, but none matched:

../../../src/test/server/server_session_test.cpp:209: tried expectation #0: EXPECT_CALL(*_connection, send_ready_for_query())...
         Expected: the expectation is active
           Actual: it is retired
         Expected: to be called once
           Actual: called once - saturated and retired
../../../src/test/server/server_session_test.cpp:242: tried expectation #1: EXPECT_CALL(*_connection, send_ready_for_query())...
         Expected: all pre-requisites are satisfied
           Actual: the following immediate pre-requisites are not satisfied:
../../../src/test/server/server_session_test.cpp:239: pre-requisite #0
                   (end of pre-requisites)
         Expected: to be called once
           Actual: never called - unsatisfied and active
unknown file: Failure

Unexpected mock function call - taking default action specified at:
../../../src/test/server/server_session_test.cpp:54:
    Function call: receive_packet_header()
          Returns: 16-byte object <F0-CF 8C-09 00-00 00-00 F0-19 10-0A 00-00 00-00>
Google Mock tried the following 2 expectations, but none matched:

../../../src/test/server/server_session_test.cpp:213: tried expectation #0: EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: the expectation is active
           Actual: it is retired
         Expected: to be called once
           Actual: called once - saturated and retired
../../../src/test/server/server_session_test.cpp:243: tried expectation #1: EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: all pre-requisites are satisfied
           Actual: the following immediate pre-requisites are not satisfied:
../../../src/test/server/server_session_test.cpp:242: pre-requisite #0
                   (end of pre-requisites)
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:236: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_command_complete(_))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:233: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_data_row(_))...
         Expected: to be called 3 times
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:230: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_row_description(_))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:239: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_notice(_))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:242: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_ready_for_query())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:243: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
[  FAILED  ] ServerSessionTest.SessionExecutesSimpleQueryCommand (2 ms)
[ RUN      ] ServerSessionTest.SessionPerformsStartup
[       OK ] ServerSessionTest.SessionPerformsStartup (1 ms)
[ RUN      ] ServerSessionTest.SessionShutsDownOnErrorReceivingPacketHeader
Some connection problem
[       OK ] ServerSessionTest.SessionShutsDownOnErrorReceivingPacketHeader (1 ms)
[ RUN      ] ServerSessionTest.SessionSendsErrorWhenBindingUnknownNamedStatement
[       OK ] ServerSessionTest.SessionSendsErrorWhenBindingUnknownNamedStatement (1 ms)
[ RUN      ] ServerSessionTest.SessionHandlesExtendedProtocolFlow
unknown file: Failure
C++ exception with description "../../../src/lib/storage/storage_manager.cpp:43 No such table named 'table_a'" thrown in the test body.
../../../src/test/server/server_session_test.cpp:297: Failure
Actual function call count doesn't match EXPECT_CALL(*_task_runner, dispatch_server_task(An<std::shared_ptr<ExecuteServerPreparedStatementTask>>()))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:266: Failure
Actual function call count doesn't match EXPECT_CALL(*_task_runner, dispatch_server_task(An<std::shared_ptr<CreatePipelineTask>>()))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:283: Failure
Actual function call count doesn't match EXPECT_CALL(*_task_runner, dispatch_server_task(An<std::shared_ptr<BindServerPreparedStatementTask>>()))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:269: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_status_message(NetworkMessageType::ParseComplete))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:286: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_status_message(NetworkMessageType::BindComplete))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:252: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, send_ready_for_query())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:293: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_execute_packet_body(42))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:276: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_bind_packet_body(42))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:259: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_parse_packet_body(42))...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:256: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:273: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
../../../src/test/server/server_session_test.cpp:290: Failure
Actual function call count doesn't match EXPECT_CALL(*_connection, receive_packet_header())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
[  FAILED  ] ServerSessionTest.SessionHandlesExtendedProtocolFlow (1 ms)
```

**Comment by mrks** [comment]
Haven't seen this for a while now. The network fails are a different issue. Closing for now.
